server/ice_servers/coturn.py

```diff
@@ -12,15 +12,23 @@
 from hashlib import sha1
 from typing import Dict, List
 
-from server.config import COTURN_HOSTS, COTURN_KEYS, TWILIO_TTL
+from server.config import config
 
 
 class CoturnHMAC:
-    def __init__(self, coturn_hosts=COTURN_HOSTS, coturn_keys=COTURN_KEYS):
+    def __init__(self, coturn_hosts=None, coturn_keys=None):
+        if coturn_hosts is None:
+            coturn_hosts = config.COTURN_HOSTS
+        if coturn_keys is None:
+            coturn_keys = config.COTURN_KEYS
```

Comment:
Syntax would be nicer with e.g. `self.coturn_hosts = coturn_hosts or config.COTURN_HOSTS`, but I don't like how that would handle e.g. `coturn_hosts = []`.
`self.coturn_hosts = coturn_hosts if coturn_hosts is not None else config.COTURN_HOSTS` on the other hand is very mouthy.

Reply:
Probably better to default to an empty list and pass in the variables form config explicitly when created.